packages/snfoundry/scripts-ts/verify-contracts.ts

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+import { execSync } from 'child_process';
+import yargs from 'yargs';
+
+// Import deployedContracts
+const deployedContractsPath = path.resolve(__dirname, '../../nextjs/contracts/deployedContracts');
+const deployedContracts = require(deployedContractsPath).default;
```

Comment:
Thanks for the PR! Wondering why do we need `require` and `import` at the same time? I think it's not a good practice (usually linter will complain too).

Reply:
Ready applied, everything working correctly.